Framework/Algorithms/src/RenameWorkspace.cpp

```diff
@@ -175,6 +180,8 @@ bool RenameWorkspace::processGroups() {
             this->name(), this->version());
         alg->setPropertyValue("InputWorkspace", names[i]);
         alg->setPropertyValue("OutputWorkspace", wsName);
+        alg->setChild(true);
+        alg->enableHistoryRecordingForChild(false);
```

Comment:
Yes I do, however it is a bit out of the scope of this PR in my opinion, plus it depends on how they call the child algorithms on the groups I think I tested this with ReflectometryReductionOneAuto as well and the history was still complete I believe. It would have to be on an algorithm by algorithm basis.

Reply:
I agree.